security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -200,12 +204,17 @@ func (s *sdsservice) Close() {
 }
 
 // toEnvoySecret converts a security.SecretItem to an Envoy tls.Secret
-func toEnvoySecret(s *security.SecretItem) *tls.Secret {
+func toEnvoySecret(s *security.SecretItem, caRootPath string) *tls.Secret {
 	secret := &tls.Secret{
 		Name: s.ResourceName,
 	}
-
-	cfg, ok := model.SdsCertificateConfigFromResourceName(s.ResourceName)
+	cfg := nodeagentutil.SdsCertificateConfig{}
+	ok := false
+	if s.ResourceName == "file-root:system" {
```

Comment:
Let's make this a constant that is referenced across. Magic strings can be hard to debug.

Reply:
Constant is in `pkg/security/security.go` with other constants